src/cache.cpp

```diff
@@ -55,30 +59,29 @@ namespace {
 	cache_effect_type cache_effects;
 
 	std::string system_name;
-	BitmapRef default_system;
 
 	std::string system2_name;
 
 	constexpr int cache_limit = 10 * 1024 * 1024;
 	size_t cache_size = 0;
 
 	void FreeBitmapMemory() {
-		int32_t cur_ticks = DisplayUi->GetTicks();
+		auto cur_ticks = clock::now();
 
 		for (auto& i : cache) {
 			if (i.second.bitmap.use_count() != 1) {
 				// Bitmap is referenced
 				continue;
 			}
 
-			if (cache_size <= cache_limit && cur_ticks - i.second.last_access < 3000) {
+			if (cache_size <= cache_limit && cur_ticks - i.second.last_access < 3000s) {
```

Comment:
is this correct? The original check is for 3000ms/3s

Reply:
yeah, he force-pushed this away. seems like an oversight.

Reply:
Was a bug on my part, fixed.